testing/camunda-process-test-java/src/main/java/io/camunda/process/test/impl/client/CamundaApiClient.java

```diff
@@ -35,8 +35,10 @@ public class CamundaApiClient {
 
   // Operate v1 endpoints
   private static final String PROCESS_INSTANCE_GET_ENDPOINT = "/v1/process-instances/%d";
+  private static final String PROCESS_INSTANCE_SEARCH_ENDPOINT = "/v1/process-instances/search";
```

Comment:
❌  Why `v1` ? It should be `v2`. If I'm not wrong the search endpoint are available only under `v2`. Can we also update the other path to use the new API version (not sure if it's going to be a breaking change)

Reply:
Currently, CPT uses the Operate endpoint (v1): https://docs.camunda.io/docs/next/apis-tools/operate-api/specifications/search-1/.

When the new query API (v2) is working, I would love to switch to the new API (v2) and use the SDK. However, the auth issue is blocking us at the moment. 

For now, let's stay at the old v1 API. :sweat_smile: